code/api/src/modules/user/resolvers.js

```diff
@@ -19,7 +19,13 @@ export async function create(parentValue, { name, email, password }) {
     return await models.User.create({
       name,
       email,
-      password: passwordHashed
+      password: passwordHashed,
+      bio,
+      image,
+      street,
+      city,
+      state,
+      zip
     })
   } else {
     // User exists
```

Comment:
Ooooooh What's a resolver?

Reply:
You will soon find out! ;)

Reply:
Abstracted file from the query and resolvers to make Sequelize queries to db models to retrieve data! Did an explanation over zoom